components/InfoBox/index.scss

```diff
@@ -0,0 +1,36 @@
+@import '~kununu-theme/scss/base/variables';
+
+.infoBox {
+  position: absolute;
+  display: block;
+  z-index: 100;
+  width: 100vw;
```

Comment:
Is supported since IE9, only other viewport units (like vmax, vmin) are unsupported in IE right now